clients/drcachesim/simulator/caching_device_stats.h

```diff
@@ -64,9 +67,88 @@ enum class metric_name_t {
     FLUSHES
 };
 
+struct bound {
+    addr_t beg;
+    addr_t end;
+};
+
+class access_count_t {
+public:
+    access_count_t(int block_size)
+        : block_size_(block_size)
+    {
+        if (!IS_POWER_OF_2(block_size)) {
+            ERRMSG("Block size should be a power of 2.");
+            return;
+        }
+        int block_size_bits = compute_log2(block_size);
+        block_size_mask_ = ~((1 << block_size_bits) - 1);
+    }
+
+    // Takes non-alligned address and inserts bound consisting of the nearest multiples
```

Comment:
spelling: "alligned"

Reply:
(The convention is to reply "Done" or somesuch and mark as Resolved)

Reply:
Sure, I will remember